### PR TITLE
Add MySQL Airflow extractor

### PR DESCRIPTION
**Pull request by nicosuave**
Will update with checks for the `MySQLOperator`, as per https://github.com/MarquezProject/marquez/pull/1414, as well as sort dependencies.

### CODE REVIEW

**Comment by mobuchowski** [comment]
@nicosuave seems like mysqlclient requires native client library: **libmysqlclient-dev** 

Can you add step installing this to [test Dockerfile](https://github.com/MarquezProject/marquez/blob/main/integrations/airflow/Dockerfile.tests)?

File: integrations/airflow/Dockerfile.tests [hunk 1]
```diff
@@ -1,4 +1,6 @@
 FROM python:3.6-slim
+RUN apt-get update && \
+    apt-get install -y libmariadb-dev
```
Comment (mobuchowski) [inline integrations/airflow/Dockerfile.tests:3]: I think we need `libmariadbclient-dev` here

**Comment by mobuchowski** [comment]
So, tried to compile that myself, and it seems to work with 
```RUN apt update && apt install -y build-essential gcc libmariadb-dev-compat```

Although, at this point I'd refrain from adding it to test dependencies - it would just sure that the base image build, the actual mysql integration isn't integration tested. Could you manually test that extractor? 

I'm working at refactoring integrations' integration tests - after that, adding integration test for this would be much simpler.

**Comment by nicosuave** [comment]
Thanks @mobuchowski 🙌 , will dive back into this over my weekend

**Comment by wslulciuc** [comment]
Hey, @nicosuave 👋. I was wondering if you had any questions on the MySql extractor implementation? And, as a side note, since this PR was opened, we've donated the `marquez-airflow` integration to [`OpenLineage`](https://github.com/OpenLineage/OpenLineage). I'd be great to have this extractor as part of OpenLineage and was wondering if you could re-open this PR against OpenLineage repo?

**Comment by nicosuave** [comment]
> Hey, @nicosuave 👋. I was wondering if you had any questions on the MySql extractor implementation? And, as a side note, since this PR was opened, we've donated the `marquez-airflow` integration to [`OpenLineage`](https://github.com/OpenLineage/OpenLineage) (and will soon be deprecated as of `0.17.0`). I'd be great to have this extractor as part of OpenLineage and was wondering if you could re-open this PR against OpenLineage repo?

@wslulciuc My apologies I've not gotten around to this— closing this PR now and will re-open in OpenLineage after I've tested following suggestions from @mobuchowski. Thanks for keeping me in the loop on the direction of the project 🙏.

**Pull request closed without merging**